README.md

```diff
@@ -1,4 +1,6 @@
 ![GitHub forks](https://img.shields.io/github/forks/beyond-io/picATrip?style=social) ![GitHub issues](https://img.shields.io/github/issues-raw/beyond-io/picATrip?logo=GitHub) ![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/beyond-io/picATrip?logo=GitHub) ![GitHub top language](https://img.shields.io/github/languages/top/beyond-io/picATrip)
+[![Black](https://github.com/beyond-io/PicATrip/actions/workflows/black.yml/badge.svg)](https://github.com/beyond-io/PicATrip/actions/workflows/black.yml)
+[![Flake8](https://github.com/beyond-io/PicATrip/actions/workflows/flake8.yml/badge.svg)](https://github.com/beyond-io/PicATrip/actions/workflows/flake8.yml)
```

Comment:
I think it will look better if you won't have a separate line (new line) at least between the black and the flake8 badges, maybe considering doing that also between the other badges and the black badge, if we want it all to be in the same line (see the badges already here, no new lines in between the badges..)

Reply:
I don't sure I'm following. This is what I suggest:
![image](https://user-images.githubusercontent.com/72739568/117368914-d7ae0700-aecc-11eb-903f-a0d1adcec1ba.png)

Do you want to add a new line like that?
![image](https://user-images.githubusercontent.com/72739568/117369119-22c81a00-aecd-11eb-8d47-1197f7320314.png)